### ISSUE TITLE
Evaluate whether setting host memory limits is still necessary or beneficial

### ISSUE THREAD

**Issue by randerzander** [rapidsai/gpu-bdb#167]
We currently [set host memory limits to the max available system memory](https://github.com/rapidsai/gpu-bdb/blob/main/tpcx_bb/cluster_configuration/cluster-startup-slurm.sh#L9).

As @beckernick  suggests [here](https://github.com/rapidsai/dask-cuda/issues/169#issuecomment-653050853), the many Dask improvements to GPU memory management may have made the host memory configuration unnecessary. We should evaluate if that's true.

**Comment by beckernick** [comment]
I agree, we should test this. My expectation is that the net effect of a variety changes makes this primarily relevant for conducting outsized workloads on small infrastructure. Given that isn't likely the default, we likely don't need to default to max system memory.
